src/public/serviceHostKeys.ts

```diff
@@ -64,9 +43,34 @@ class ServiceElement {
         <h4>${this.service.name}</h4>
         <li class="list-group-item">
           ${serviceKeys}
-          <button type="button" class="btn btn-primary">Create</button>
+          <button type="button" class="btn btn-primary" id="submitKeys">Create</button>
```

Comment:
don't use Ids. You can have many providers. each page should only have 1 element with id attribute, so don't use ids